src/rnp/rnpcfg.c

```diff
@@ -116,8 +118,12 @@ rnp_cfg_find(rnp_cfg_t *cfg, const char *key)
     return (i == cfg->count) ? -1 : (int) i;
 }
 
-/* resize keys/vals arrays to the new size. Only expanding is supported */
-static int
+/** @brief resize keys/vals arrays to the new size. Only expanding is supported now.
```

Comment:
Idea: So why not too call it ``rnp_cfg_expand``. To support shrinking you probably will need change function definition anyway, isn't?